src/components/pickers/UserRolesPicker.js

```diff
@@ -32,6 +32,9 @@ const UserRolesPicker = ({
     { str: searchString },
   );
 
+  const roles = data?.role?.edges.map((edge) => edge.node) ?? [];
+  const uniqueValues = [...new Map(value?.map((role) => [role.isSystem, role])).values()];
```

Comment:
`isSystem` value determines that given role is built into openIMIS system (schema admin, CA, etc.). Administrators can add custom roles that have `isSystem` set to null. Witch this check, custom roles would be treated as duplicates. Can we change this to other property (e.g. id)?

Reply:
Yes, you are right. I changed `isSystem` to the `id` property.

Reply:
Thanks